src/components/SeatsAnalysisViewer.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+    <div class="tiles-container">      
+        <v-card elevation="4" color="white" variant="elevated" class="mx-auto my-3" style="width: 300px; height: 175px;">
+            <v-card-item>
+                <div>
+                <div class="text-overline mb-1" style="visibility: hidden;">filler</div>
+                <div class="text-h6 mb-1">Total Assigned  </div>
+                <div class="text-caption">
+                    currently assigned seats
```

Comment:
```suggestion
                    currently assigned seats
```Currently assigned seats